types/plotly.js/lib/locales/zh-tw.d.ts

```diff
@@ -0,0 +1,3 @@
+import { Locale } from '../core';
+
+export default Locale;
```

Comment:
This needs `export =` instead of `export default`. Refer to https://github.com/DefinitelyTyped/DefinitelyTyped#a-package-uses-export--but-i-prefer-to-use-default-imports-can-i-change-export--to-export-default

```js
const x = require("plotly.js/lib/locales/af");
console.log(x.default);
```
```
> node a.js
undefined
```

Reply:
I changed it, thank you.